careerfairsystems/arkad-app-2018
# Fix overview map click on Android

## Change description

Fixes the issue with the Android overview map not being clickable.
- Removed the zoom package
- Updated SVG package version
  - Needed to update gradle version for Android
- Loading the house map at a later time so no delay is experienced when pressing a building

## How to verify

Run the app on both Android and iOS and confirm that everything still works (especially the overview map).

## Issues fixed

This fixes #123.
